lib/modules/csv_exporter.rb

```diff
@@ -80,10 +80,14 @@ def self.format_customised_questions(response, default=@@default_na)
         customised_question_row << text << options << answer
       end
 
-    if customised_question_row.length == 3
+    if customised_question_row.length == 6
       customised_question_row << ["n/a", "n/a", "n/a"]
     end
 
+    if customised_question_row.length == 3
+      customised_question_row << ["n/a", "n/a", "n/a", "n/a", "n/a", "n/a"]
+    end
+
```

Comment:
The download is working well and is using semi-colons! Would it be possible here to add however 'n/a's are needed to make the length of the array 9 at the end, rather than using these if cases?

Reply:
Good point!